prisma/migrations/20220216104243_add_auditing_to_workflow_table/migration.sql

```diff
@@ -0,0 +1,35 @@
+-- Add Workflow_Audit table along with trigger for this table
+CREATE TABLE "Workflow_Audit"
+(
+    operation char(20) NOT NULL,
+    timestamp timestamp DEFAULT CURRENT_TIMESTAMP,
+    id        text     NOT NULL,
+    before    JSONB    NOT NULL,
+    after     JSONB    NOT NULL
+);
+
+CREATE OR REPLACE FUNCTION Workflow_Audit_Trigger() RETURNS TRIGGER AS
+$Workflow_Audit$
+BEGIN
+    IF (TG_OP = 'DELETE') THEN
+        INSERT INTO "Workflow_Audit" (operation, id, before, after)
+        VALUES ('DELETE', old.id, TO_JSONB(old), '"null"');
+    ELSIF (TG_OP = 'INSERT') THEN
+        INSERT INTO "Workflow_Audit" (operation, id, before, after)
+        VALUES ('INSERT', new.id, '"null"', TO_JSONB(new));
+    ELSIF (TG_OP = 'UPDATE') THEN
+        old."updatedAt" = new."updatedAt";
+        IF OLD IS DISTINCT FROM NEW THEN
```

Comment:
I take it none of these tables have existing audit type columns that would trigger this unnecessarily like you mentioned happened with Person table?